utilities/cloudharness_utilities/deployment-configuration/helm/templates/auto-database.yaml

```diff
@@ -46,27 +60,24 @@ spec:
         volumeMounts:
           - name: {{ .app.harness.database.name | quote }}
             mountPath: /data/db
+          {{- if .root.Values.backup.active }}
+          - name: "db-backup"
+            mountPath: /backup
```

Comment:
hmm, this way a single DB container would have access to backups of all DBs, I think we should limit this somehow

Reply:
I'm not a privacy/security expert but I don's see backups to be visible to other applications as a problem. I think it's just handy to have all backups in the same place to look at all system backup or copy all backups to a different place.

Reply:
agree with @filippomc , security wise, if a "hacker" has access to the cluster he/she can access all the volumes already